### PR TITLE
New, simpler tool for splitting intervals and changes to M2 wdl.

### PR DESCRIPTION
**Pull request by davidbenjamin**
@LeeTL1220 Here you go.  The wdl changes are tested, with and without intervals specified.

### CODE REVIEW

File: scripts/mutect2_wdl/mutect2.wdl [hunk 1]
```diff
@@ -413,7 +407,7 @@ task SplitIntervals {
   }
 
   output {
-    Array[File] interval_files = glob("*.interval_list")
+    Array[File] interval_files = glob("*.intervals")
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: This extension is not going to cause any issues, right?

**Comment by LeeTL1220** [comment]
@davidbenjamin Back at you.  Not much here.  

*I'm assuming that bed and interval_list are tested elsewhere.*

File: scripts/mutect2_wdl/mutect2.wdl (hunk 1 above)
Comment (davidbenjamin) [inline scripts/mutect2_wdl/mutect2.wdl]: Nope, it works.

**Comment by davidbenjamin** [comment]
@LeeTL1220 I have tested bed and interval_list inputs.

**Comment by LeeTL1220** [comment]
Great.  Merge away!

**Pull request merged**